### ISSUE TITLE
.yara sigs removed from database

### ISSUE THREAD

**Issue by smwardle** [extremeshok/clamav-unofficial-sigs#99]
New in version 5.2.0:
Sanesecurity_sigtest.yara and Sanesecurity_spam.yara  were removed from clam database.

master.conf has them included along with
yararulesproject_enabled="yes" and enable_yararules="yes"

**Comment by extremeshok** [comment]
post the output of "clamscan -V"

**Comment by extremeshok** [comment]
Fixed, pushing the small update

````
ls -laFh /var/lib/clamav-unofficial-sigs/dbs-ss | grep yar
-rw-r--r--   1 clamav clamav 1.5K Jul  1  2015 Sanesecurity_sigtest.yara
-rw-r--r--   1 clamav clamav   72 Jul  1  2015 Sanesecurity_sigtest.yara.sig
-rw-r--r--   1 clamav clamav 1.3K Feb 22 12:21 Sanesecurity_spam.yara
-rw-r--r--   1 clamav clamav   72 Feb 22 12:54 Sanesecurity_spam.yara.sig
-rw-r--r--   1 clamav clamav 1.6K Apr 16 00:45 winnow_malware.yara
-rw-r--r--   1 clamav clamav   72 Apr 16 00:45 winnow_malware.yara.sig
````

**Comment by extremeshok** [comment]
fixed in 5.2.1
